src/components/ec-btn/ec-btn.spec.ts

```diff
@@ -1,26 +1,38 @@
+import type { MountingOptions } from '@vue/test-utils';
+// eslint-disable-next-line import/no-extraneous-dependencies
 import { mount } from '@vue/test-utils';
+import type { RouteLocationNamedRaw } from 'vue-router';
 
+import type { CVueWrapper } from '../../../tests/utils/global';
+import { IconName } from '../ec-icon/iconNames';
 import EcBtn from './ec-btn.vue';
+import type { ButtonProps } from './types';
+import { ButtonCategory, ButtonSize } from './types';
 
 describe('EcBtn', () => {
-  function mountBtn(props, mountOpts) {
-    return mount(EcBtn, {
-      props,
-      ...mountOpts,
-    });
+  function mountBtn(props?: ButtonProps, mountOpts?: MountingOptions<ButtonProps>) {
+    return mount(EcBtn as any,  // eslint-disable-line
+      {
+        props,
+        ...mountOpts,
+      },
+    ) as CVueWrapper;
   }
 
   it('should render a <button> element by default', () => {
-    const wrapper = mountBtn();
+    const wrapper = mountBtn({
+      category: ButtonCategory.Primary,
```

Comment:
We shouldn't change the tests while doing migrations. Why is this needed?

Reply:
Because category is a required prop.

Reply:
It wasn't required before, we should fix that then: https://github.com/Ebury/chameleon/blob/master/src/components/ec-btn/ec-btn.vue#L108-L113

Reply:
My bad, I'll fix it. Thanks!